src/components/transaction/DataFormat.vue

```diff
@@ -101,34 +115,41 @@ export default defineComponent({
             isOverflowing,
             showOverflow,
             toggleOverflow,
-            maxHeight,
-            switchHeight,
-            useColor,
+            maxHeightStyle,
         };
     },
 });
 </script>
 
 <template>
-
-<div class="relative-position" :class="showOverflow ? '' : 'overflow-hidden'" :style=" (showOverflow || !isOverflowing) ? '' : `max-height: calc(${maxHeight}px - ${switchHeight}px)`">
-  <div class="row" v-if="actionName === 'transfer'" ref="dataBox">
-    <div class="col-12"><span class="text-bold">
-        <AccountFormat :account="transferData.from" type="account"></AccountFormat></span><span class="text-bold">&nbsp; → &nbsp;
-        <AccountFormat :account="transferData.to" type="account">&nbsp;</AccountFormat></span><span class="text-bold">{{ ' ' + transferData.quantity}}</span></div>
-    <div class="col-12">
-      <div class="text-weight-bold">memo:&nbsp;<span class="text-weight-regular" v-if="transferData.memo">{{transferData.memo}}</span><span class="text-weight-regular" v-else>n/a</span></div>
+<div
+    class="relative-position"
+    :class="{'overflow-hidden': !showOverflow, 'div-compressed': !showOverflow}"
+>
+    <div v-if="actionName === 'transfer'" ref="dataBox" class="row">
+        <div class="col-12">
+            <span class="text-bold">
+                <AccountFormat :account="transferData.from" type="account"/></span><span class="text-bold">&nbsp; → &nbsp;
+                <AccountFormat :account="transferData.to" type="account">&nbsp;</AccountFormat></span><span class="text-bold">{{ ' ' + transferData.quantity }}</span>
+        </div>
+        <div class="col-12">
+            <div class="text-weight-bold">
+                memo:&nbsp;<span v-if="transferData.memo" class="text-weight-regular">{{ transferData.memo }}</span><span v-else class="text-weight-regular">n/a</span>
+            </div>
+        </div>
     </div>
-  </div>
-  <div class="row" v-else ref="dataBox">
-    <div class="col-12" v-for="val in formatGeneralData(data)" :key="val.key">
-      <div class="text-weight-bold">{{val.key}} :<span class="text-weight-regular" v-if="isAccount(val.key)">
-          <AccountFormat :account="val.value" type="account">&nbsp;</AccountFormat></span><span class="text-weight-regular" v-else>{{val.value}} &nbsp;</span></div>
+    <div v-else ref="dataBox" class="row">
+        <PrettyPayload :depth="0" :payload="formatGeneralData(actionData)"/>
```

Comment:
If you look in console you see a bunch of `runtime-core.esm-bundler.js:38 [Vue warn]: Invalid prop: type check failed for prop "payload". Expected Object, got Null  
  at <PrettyPayload depth=2 payload=null > ` warnings. This is because the payload property for the `PrettyPayload` component is null in some cases. Updating `payload` property in src/components/transaction/PrettyPayload.vue to accept null as an optional type should resolve those e.g. `payload: { type: Object || null, required: true },
`